startleft/transformer.py

```diff
@@ -37,13 +40,20 @@ def transform_trustzones(self):
                 if "$source" in mapping and isinstance(mapping["$source"], dict):
                     if "$singleton" in mapping["$source"] and trustzone_number > 0:
                         continue
+                logger.debug(f"Added trustzone: [{trustzone_element['name']}][{trustzone_element['id']}]")
```

Comment:
I'd change Added by Adding. In case of exception may be confused the message "Added" when the current trustzone is not finally added due to the exception.

Reply:
I would not change the word because the purpose of this log is to see that the trustzone was added successfully. What I'd do is move the log and the increment statements just after the added trustzone statement, although what this statement really does at the end is simply add the trustzone to an array and it should not failed... but just to be sure we can move it after.

Reply:
Ok. You're right. It's ok as is.